<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="jog_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>561</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>391</width>
       <height>315</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : rgb(235, 235, 235)</string>
     </property>
     <property name="title">
      <string>JOG Controls</string>
     </property>
     <widget class="QGroupBox" name="steps_groupbox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>100</width>
        <height>280</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>STEPS</string>
      </property>
      <widget class="QPushButton" name="step1_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>10 mm</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="step2_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>85</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>5 mm</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="step3_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>155</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>1 mm</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="step4_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>80</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>0.1 mm</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>19</y>
        <width>240</width>
        <height>281</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(235, 235, 235)</string>
      </property>
      <property name="title">
       <string>XY Axis</string>
      </property>
      <widget class="QPushButton" name="yplus_btn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>81</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Y +</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="xminus_btn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>81</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>X -</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="yminus_btn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>190</y>
         <width>81</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Y -</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="xplus_btn">
       <property name="geometry">
        <rect>
         <x>149</x>
         <y>110</y>
         <width>81</width>
         <height>60</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>X +</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="home_groupbox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>110</width>
       <height>315</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : rgb(235, 235, 235)</string>
     </property>
     <property name="title">
      <string>Home</string>
     </property>
     <widget class="QPushButton" name="xhome_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>80</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>X Home</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="yhome_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>80</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Y Home</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="home_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>80</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="com_box">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>181</width>
       <height>180</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color : rgb(235, 235, 235)</string>
     </property>
     <property name="title">
      <string>COM Setting</string>
     </property>
     <widget class="QComboBox" name="port_combo">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="baud_combo">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="port_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>55</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLabel" name="baudrate_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Baudrate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connect_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="update_btn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>201</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 21, 56);
alternate-background-color: rgb(255, 85, 127);</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
